migrations/20200804025157-create-models.js

```diff
@@ -0,0 +1,308 @@
+"use strict";
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    await queryInterface.sequelize.query(`
+      -- ER/Studio Data Architect SQL Code Generation
+      -- Project :      BATS Logical Data Model.DM1
+      --
+      -- Date Created : Sunday, August 02, 2020 18:13:09
+      -- Target DBMS : PostgreSQL 9.x
+      --
+      -- 
+      -- TABLE: ambulance 
+      --
+      CREATE TABLE ambulance(
+          ambulance_uuid                          uuid           NOT NULL,
+          emergencymedicalserviceprovider_uuid    uuid           NOT NULL,
+          ambulanceidentifier                     varchar(50)    NOT NULL,
+          recordcreatetimestamp                   timestamp      NOT NULL,
```

Comment:
@julianmclain just a note that Sequelize has built-in support for setting created/updated timestamps when creating/saving records using the ORM. The default naming convention is different, so you'll want to customize your model to match these column names as described here:

https://sequelize.org/master/manual/model-basics.html#timestamps

Reply:
What's your opinion on removing our timestamps and using the built-in ones? I saw in the documentation:
```
Note: This is done in the Sequelize level (i.e. not done with SQL triggers). This means that direct SQL queries (for example queries performed without Sequelize by any other means) will not cause these fields to be updated automatically.
```

I think it would be nice to have the timestamps just update automatically.